app/src/main/res/values/strings.xml

```diff
@@ -168,7 +168,7 @@
     <string name="desc_filedic">file</string>
     <string name="desc_other">other</string>
     <string name="msg_setgoogle">Set Google search engine now?</string>
-    <string name="action_mark">Mark as AD</string>
+    <string name="action_mark">Mark as Ad</string>
```

Comment:
Maybe you could change this to Mark as an AD

Reply:
Why should we spell it in all capital letters?

Reply:
AD is a typo on my end. I mean could you add an "an". It still sounds snappy.

Reply:
Like the verb be, the article is always omitted, too. Hope you agree.